src/pgs_util.h

```diff
@@ -1,5 +1,6 @@
 #ifndef __PGS_UTIL_H__
 #define __PGS_UTIL_H__
+#include <c.h>
```

Comment:
Hmm. What is this for? Is this to get the `float8` typedef?

Reply:
I would propose to use double type.